kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -602,10 +533,7 @@ private Optional<Long> getStartCheckpointVersion(Engine engine, Optional<Long> v
                       });
             })
         .orElseGet(
-            () -> {
-              logger.info("Loading last checkpoint from the _last_checkpoint file");
```

Comment:
The invoker of a method should only log additional context that the invokee would not know about.

In this case, the invokee should be responsible for printing this.